genie-web/src/main/java/com/netflix/genie/web/configs/PropertiesConfig.java

```diff
@@ -77,4 +79,18 @@ public HealthProperties healthProperties() {
     public S3FileTransferProperties s3FileTransferProperties() {
         return new S3FileTransferProperties();
     }
+
+    /**
+     * Subset of properties concerned with per-user number of active jobs.
+     * Notice the 'prototype' scope. If this bean is injected via {@code javax.inject.Provider} or Spring's
```

Comment:
Better to use `@link` here instead of `@code` so the javadocs are linked? not sure.